builds/android/transifex_helper.rb

```diff
@@ -0,0 +1,112 @@
+#!/bin/ruby
+#
+# joins the text files used by fastlane/supply to a XML file
+# which can be uploaded to transifex or splits this file.
+# by carstene1ns 2017, released under the MIT license
+
+# these libraries are provided by ruby, so no external dependencies
+require "rexml/document"
+require 'optparse'
+
+# default options
+metadata_dir = "metadata"
+lang_code = "en-US"
+mode = ""
+
+optparse = OptionParser.new do |opts|
+  opts.banner = "Usage: transifex_helper.rb [options]"
+  opts.separator ''
+  opts.on("-m", "--mode MODE", "Set operation mode: split, join", String) do |m|
```

Comment:
The documentation for split and join could be improved. Looks like join takes the txt files and writes them in a XML and split is the inverse operation.